helix-term/src/keymap.rs

```diff
@@ -502,6 +502,7 @@ impl Default for Keymaps {
             },
 
             "\"" => select_register,
+            "C-z" => suspend,
```

Comment:
Document this in the `book/`

Reply:
I don't think this is something that we need to explicitly document. Users will expect this to work since all cli is expected to use ctrl-z for this sort of event. I did look into that but not sure which category to put last time.